test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -50,7 +50,7 @@ TEST(nntrainer_nnmodel_construct_deconstruct, nntrainer_02_n) {
  * @brief Neural Network Model Construct wit Configuration File Test
  * @return 0 success, -EINVAL if failed.
  */
-TEST(nntrainer_nnmodel_construct_deconstruct, nntrainer_04_n) {
+TEST(nntrainer_nnmodel_construct_deconstruct, nntrainer_03_n) {
```

Comment:
The test name seems to be meaningless.
How about changing it like this?
`TEST(nntrainer_tizen_capi, construct_deconstruct_xx_x) `

Reply:
That would be good. Thanks.